modules/op-release-notes-1-7.adoc

```diff
@@ -27,7 +27,7 @@ The `preview` and `stable` channels will be deprecated and removed in a future r
 ** To add the annotations to a specific namespace, export the cluster task as a task to that namespace.
 // https://issues.redhat.com/browse/SRVKP-1514
 
-* With this update, the `when` expressions in a `Task` object are scoped to guard the tasks by default. To continue guarding the `Task` object and its dependent tasks, set the `scope-when-expressions-to-task` flag to `true`.
+* Before this update, the `when` expression would skip a `Task` object and its dependent tasks when certain conditions were not met. With this update, you can scope the `when` expression to guard the `Task` object only, not its dependent tasks. To enable this update, set the `scope-when-expressions-to-task` flag to `true` in the `TektonConfig` CRD.
```

Comment:
Maybe, for clarity:
```suggestion
* Before this update, if certain conditions were not met, the `when` expression skipped a `Task` object and its dependent tasks. With this update, you can scope the `when` expression to guard the `Task` object only, not its dependent tasks. To enable this update, set the `scope-when-expressions-to-task` flag to `true` in the `TektonConfig` CRD.
```
- Follows the "if...when" pattern: puts the triggering condition first.
- Avoids repeating different instances of "when" in the same sentence.

Reply:
Incorporated this suggestion @rolfedh